llmware/configs.py

```diff
@@ -45,6 +45,7 @@ class LLMWareConfig:
              "collection_db": "mongo",
              "milvus_host": os.environ.get("MILVUS_HOST","localhost"),
              "milvus_port": int(os.environ.get("MILVUS_PORT",19530)),
+             "milvus_db": os.environ.get("MILVUS_DB", "default"),
```

Comment:
This will work , but I am wondering once we set MILVUS_DB to a particular database say 'my_db' in environment, then both tests as well as main program will use the same setting, this will lead to tests deleting the 'my_db' again. I earlier raised a pull request which passes database name as a parameter. Approach in this PR is obviously cleaner, but perhaps it has to address the deletion of 'default' database issue.